docs/debugging.md

```diff
@@ -5,3 +5,5 @@
 You can use any other regex inplace of `ContractKey.*`, it matches with debug format of key.
 
 - To log all api requests that server `RUST_LOG="fedimint_server::request"`
+
+- [Inspect and manipulate the database using `dbtool`](../fedimint-dbtool/README.md) (low level, see `fedimint-dbdump` for a higher-level inspection tool)
```

Comment:
Haha I actually added some documentation in my PR too :)

FYI in that PR I actually merged dbtool and dbump - feel free to keep here though and I can resolve merge conflicts

https://github.com/fedimint/fedimint/pull/1523